env.js

```diff
@@ -45,6 +44,49 @@ function config ( ) {
   env.devicestatus_collection = readENV('MONGO_DEVICESTATUS_COLLECTION', 'devicestatus');
 
   env.enable = readENV('ENABLE');
+
+  // parse defaults here, expect format `key1=value1 key2=value2` split on space, ignore commas
+  var rawdefaults = readENV('DEFAULTS');
```

Comment:
@MilosKozak the tests are failing because there is no value for `DEFAULTS` try:
```diff
-  var rawdefaults = readENV('DEFAULTS');
+  var rawdefaults = readENV('DEFAULTS', '');
```